pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchFromGitHub
+, pkgs
```

Comment:
I'm sorry that I initially missed it, but can you please list the dependencies (`ncurses`, `neovim`, ...) here rather than `pkgs`? With such a change, it's possible to replace a dependency for this package like this:

```
nvimpager.override {
  ncurses = mycustomncurses;
}
```

Reply:
I now removed `pkgs` from the arguments completely requiring the direct descendent. Please tell me if that is ok or if I should only have done it for `buildInputs` or maybe not for `stdenv`?